gorm-tools/src/main/groovy/gorm/tools/repository/GormRepo.groovy

```diff
@@ -438,6 +459,31 @@ trait GormRepo<D> implements RepoEntityErrors<D>, QueryMangoEntityApi<D> {
         }
     }
 
+    /**
+     * crate/update associations for given entity
+     *
+     * @param associatedEntityClasss class of associated entity
+     * @Param entity The entity for which associations are being created/updated
+     * @param assocList the list of data maps to create/update
+     * @return the list of created entities
+     */
+    List doAssociation(Class associatedEntityClass, D entity, List<Map> assocList) {
```

Comment:
can you add test just on doAssociation ? I see test on locations but would be god to have test on the method itself.